doc/nrf/create_application.rst

```diff
@@ -0,0 +1,216 @@
+
+The process for creating an |NCS| application depends on the development environment.
+
+.. _creating_vsc:
+
+Creating application in the |nRFVSC|
+====================================
+
+Use the following steps depending on the application type:
+
+.. tabs::
+
+   .. group-tab:: Workspace application (recommended)
+
+      To create a workspace application in the |nRFVSC|:
+
+      1. Open |VSC|.
+      #. Open the |nRFVSC|.
+      #. In the :guilabel:`Welcome View`, click the :guilabel:`Create a new application` action.
+         The New Application page opens.
+      #. Select :guilabel:`Workspace` as the application type.
+      #. Configure other options depending on what application you want to create.
+         See tooltips for more information.
+         If you want to create the most basic application, select ``zephyr/hello_world`` in the :guilabel:`Application template` field.
+      #. Click :guilabel:`Create Application`.
+
+      See the `extension documentation <Create a new application_>`_ for more information.
+
+   .. group-tab:: Freestanding application
+
+      To create a freestanding application in the |nRFVSC|:
+
+      1. Open |VSC|.
+      #. Open the |nRFVSC|.
+      #. In the :guilabel:`Welcome View`, click the :guilabel:`Create a new application` action.
+         The New Application page opens.
+      #. Select :guilabel:`Freestanding` as the application type.
+      #. Configure other options depending on what application you want to create.
+         See tooltips for more information.
+         If you want to create the most basic application, select ``zephyr/hello_world`` in the :guilabel:`Application template` field.
+      #. Click :guilabel:`Create Application`.
+
+      See the `extension documentation <Create a new application_>`_ for more information.
+
+You can now start :ref:`configuring and building <configuration_and_build>` the application.
+
+.. _creating_cmd:
+
+Creating application for use with command line
+==============================================
+
+The recommended process for command line takes advantage of Nordic Semiconductor's example application repository, similar to the example application used for :ref:`creating an application in Zephyr <zephyr:application>`.
+
+.. include:: /releases_and_maturity/developing/adding_code.rst
+   :start-after: example_app_start
+   :end-before: example_app_end
+
+.. note::
+   Nordic Semiconductor recommends using the example application repository, but you can also create the application by hand.
+   Follow the steps to :ref:`create your own application by hand <zephyr:application>` in Zephyr and :ref:`create your own manifest repository <dm_workflow_4_own_repo>` in the |NCS|.
+
+Use the following steps depending on the application type:
+
+.. tabs::
+
+   .. group-tab:: Workspace application (recommended)
+
+      To create a workspace application:
+
+      1. Open the `ncs-example-application`_ repository in your browser.
```

Comment:
@carlescufi this repo needs updated from upstream to pull the `VERSION` file in

Reply:
Jamie's PR: https://github.com/nrfconnect/sdk-zephyr/pull/1267